packages/wdio-crossbrowsertesting-service/src/launcher.js

```diff
@@ -0,0 +1,37 @@
+var cbt = require('cbt_tunnels')
```

Comment:
Can we use here import instead or at least const? 
Do we need to have cbt tunnels added as dependency?

Reply:
cbt_tunnels is added as a dependency in the package.json for the service.